inventory.ini

```diff
@@ -1,7 +1,7 @@
 [labservers]
 cal-ls ansible_host=10.109.127.125 chassis="10.109.123.236 10.109.123.181"
 bdc-ls ansible_host=10.61.67.200 chassis="10.61.67.128 10.61.67.131"
-rtp-ls ansible_host=10.61.67.200 chassis="10.28.236.32 10.28.236.36"
+rtp-ls ansible_host=10.28.238.4 chassis="10.28.236.251 10.28.236.28"
```

Comment:
You should use mintaka to dynamically update the IP addresses

Reply:
modify mintaka.py to implement one ansible var plugin to support this feature